lib/units/device/plugins/stats/index.js

```diff
@@ -0,0 +1,52 @@
+var events = require('events')
+
+var Promise = require('bluebird')
+var syrup = require('stf-syrup')
+
+var logger = require('../../../../util/logger')
+var dbapi = require('../../../../db/api')
```

Comment:
Thank you for the kind elaboration.
I'll look into reaper unit to learn further.